frontend/keyboardNav.js

```diff
@@ -209,29 +214,32 @@ function getNewSelection(dest: Dest, store: Store): ?ElementID {
     pix = pchildren.indexOf(store._parents.get(id));
   }
   if (dest === 'prevSibling') {
-    if (pix === 0) {
-      const roots = store.searchRoots || store.roots;
-      if (roots.indexOf(store.getParent(id)) > -1) {
-        return getRootSelection(dest, store, id);
-      }
-      const childId = pchildren[pix];
+    while (pix > 0) {
+      const childId = pchildren[pix - 1];
```

Comment:
This code is copy paste from below, but I run it for every index one by one, looking for previous non-empty sibling before giving up.